reference/tax_provider.yml

```diff
@@ -1295,6 +1295,6 @@ components:
     Authorization:
       type: http
       scheme: basic
-      description: 'The [HTTP Basic Authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication) (developer.mozilla.org) credentials used to authenticate each API request to the Tax Provider from the associated store; set and update `username` and `password` using the [Update a Connection](/api-reference/store-management/tax/tax-provider-connection/provider-connection-put) request.'
+      description: 'The [HTTP Basic Authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication) (developer.mozilla.org) credentials used to authenticate each API request to the Tax Provider from the associated store; set and update `username`, `password`, and optionally `profile`, using the [Update a Connection](/api-reference/store-management/tax/tax-provider-connection/provider-connection-put) request. `profile` is an optional field and will be used with supporting providers only.'
```

Comment:
Do we also need to update https://github.com/bigcommerce/api-specs/pull/1144/files#diff-93914f3f9fab6be8b0dd3d1edebf8fceb5d1fe54375f35a90de19d2cf48979cfR86?

Reply:
Good call @benpratt77 , I updated the description for Update Connection endpoint